inst/safetyGraphics_app/global.R

```diff
@@ -11,6 +11,14 @@ library(stringr)
 library(DT)
 library(haven)
 library(tidyr)
+library(shinybusy)
+
+# use metadata in user settings folder if provided
+if(!is.null(getShinyOption("settings_location"))){
```

Comment:
safetyGraphicsApp() now defaults this option to getwd() if it's not supplied, so I think we'll need to change our logic here.